test/simctl-e2e-specs.js

```diff
@@ -230,5 +230,19 @@ describe('simctl', function () {
         deviceTypes.filter((el) => el.includes('iPhone')).length.should.be.above(1);
       });
     });
+    describe('getSimctlList', function () {
+      it('should get everything from xcrun simctl list', async function () {
+        const fullList = await getSimctlList();
+        fullList.should.have.property('devicetypes');
+        fullList.should.have.property('runtimes');
+        fullList.should.have.property('devices');
+        fullList.should.have.property('pairs');
+        fullList.devicetypes.length.should.be.above(200);
```

Comment:
Travis disagrees with `200` here. It seems there to be anywhere between 32 (for Xcode 7.3) to 44 (for Xcode 10.2).